content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Service Accounts
+description: >
+  Learn about the Kubernetes ServiceAccount object.
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+This page introduces the ServiceAccount object in Kubernetes, providing
+information about how service accounts work, use cases, limitations,
+alternatives, and links to resources for additional guidance.
+
+<!-- body -->
+
+## What are service accounts? {#what-are-service-accounts}
+
+A service account is a type of non-human account that, in Kubernetes, provides
+an identity for the processes that run in Pods. Pods can use service accounts
+to make calls to the Kubernetes API server. Service accounts exist as
+ServiceAccount objects in the API server. Service accounts have the following
+properties:
+
+*  **Namespaced:** Each service account is bound to a Kubernetes
+   {{<glossary_tooltip text="namespace" term_id="namespace">}}. Every namespace
+   gets a `default` ServiceAccount upon creation.
+
+*  **Lightweight:** Service accounts exist in the cluster and use the structure
+   defined in the Kubernetes API. You can quickly create service accounts to
+   enable specific tasks.
+
+*  **Portable:** A configuration bundle for a complex containerized workload
+   might include service account definitions for the system's components. The
+   lightweight nature of service accounts and the namespaced identities make
+   the configurations portable.
+
+Service accounts are different from user accounts, which are authenticated
+human users in the cluster. User accounts don't exist in the Kubernetes API
+server; instead, the API server treats user identities as opaque data. You can
+authenticate as a user account using multiple methods.
+
```

Comment:
Optional note: some Kubernetes distributions / platforms add a custom, extension API that _does_ represent users.

Reply:
Done